### ISSUE TITLE
java.lang.IllegalArgumentException: logits must be 2-dimensional 	 [[Node: bert/encoder/layer_0/attention/self/Softmax = Softmax[T=DT_FLOAT, _output_shapes=[[1,12,128,128]], _device="/job:localhost/replica:0/task:0/device:CPU:0"](bert/encoder/layer_0/attention/self/add)]]

### ISSUE THREAD

**Issue by withme6696** [gameofdimension/java-bert-predict#2]
It runs error. 
java.lang.IllegalArgumentException: logits must be 2-dimensional
	 [[Node: bert/encoder/layer_0/attention/self/Softmax = Softmax[T=DT_FLOAT, _output_shapes=[[1,12,128,128]], _device="/job:localhost/replica:0/task:0/device:CPU:0"](bert/encoder/layer_0/attention/self/add)]]
Could you give some advice?

**Comment by gameofdimension** [comment]
提供下：
- 操作系统及版本
- tensorflow 版本
- java 版本
- multilingual_L-12_H-768_A-12.zip 的 md5

**Comment by withme6696** [comment]
win10
tensorflow 1.12
java8
md5:6C8F4DEE9C67AD37CC6830307869BBE7

**Comment by withme6696** [comment]
> 
> 
> 提供下：
> 
>     * 操作系统及版本
> 
>     * tensorflow 版本
> 
>     * java 版本
> 
>     * multilingual_L-12_H-768_A-12.zip 的 md5

已经解决了，pom中的版本不对，打扰了，谢谢
